src/asm/fstack.cpp

```diff
@@ -199,15 +224,15 @@ bool yywrap() {
 		// If this wasn't the last iteration, wrap instead of popping
 		if (fileInfoIters.front() <= context.nbReptIters) {
 			lexer_RestartRept(context.fileInfo->lineNo);
-			context.uniqueID = macro_UseNewUniqueID();
+			context.uniqueIDStr->clear(); // Invalidate the current unique ID (if any).
 			return false;
 		}
 	} else if (contextStack.size() == 1) {
 		return true;
 	}
 
-	Context oldContext = contextStack.top();
-
+	// TODO: can't we just perform all ops that require `oldContext` *before* `pop()`?
```

Comment:
This could spark a discussion in the comments about whether we actually can; not a question to be left hidden in a pushed commit.

Reply:
That would be an unrelated refactoring, so irrelevant to this PR.